src/config_test.ts

```diff
@@ -11,6 +11,7 @@ import { KubeConfig } from './config';
 import { Cluster, newClusters, newContexts, newUsers, User } from './config_types';
 
 const kcFileName = 'testdata/kubeconfig.yaml';
+const kcNoUserFileName = 'testdata/empty-user-kubeconfig.yaml';
```

Comment:
Does this file need to be added to the PR?

Reply:
added.